From 9cae7087d43e64bf7e62581fa43ac1c31a8c2acf Mon Sep 17 00:00:00 2001
From: Jared Morrow <jm@joyent.com>
Date: Tue, 19 Sep 2017 13:27:25 -0600
Subject: [PATCH] MANTA-3418 manta log uploader lockfile logic removes lockfile
 when it fails to obtain lock Reviewed by: Joshua M. Clulow <jmc@joyent.com>
 Approved by: Joshua M. Clulow <jmc@joyent.com>

---
 deps/manta-scripts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/manta-scripts b/deps/manta-scripts
index 4b86dd1..253ae31 160000
--- a/deps/manta-scripts
+++ b/deps/manta-scripts
@@ -1 +1 @@
-Subproject commit 4b86dd132d68cb177f066a83d90b8fdc9ac2cde9
+Subproject commit 253ae31df06a8b5a3d520e3874d0eb060dda4fa3
-- 
2.21.0

